main.tf

```diff
@@ -637,7 +637,7 @@ resource "helm_release" "gitlab" {
 module "monitoring" {
   source                      = "sparkfabrik/gcp-http-monitoring/sparkfabrik"
   version                     = "~> 0.4.0"
-  count                       = var.notification_channels != "" ? 1 : 0
+  count                       = trimspace(var.notification_channels[0]) != "" ? 1 : 0
   gcp_project                 = var.project_id
   uptime_monitoring_host      = var.domain
```

Comment:
var.domain represents only the domain (example.org) , not the complete FQDN (gitlab.example.org). 

Are you sure is it ok for the monitoring module?

Reply:
no, it's definitely not good.

I made a new modification.